numba/cuda/cuda_paths.py

```diff
@@ -129,6 +129,8 @@ def get_cuda_home(*subdirs):
     path.
     """
     cuda_home = os.environ.get('CUDA_HOME')
+    if cuda_home is None:
+        cuda_home = os.environ.get('CUDA_PATH')  # Try Windows CUDA installation without Anaconda
```

Comment:
@seibert Expect this line needs wrapping to 80 chars for flake8 to pass?

Reply:
good catch, I'll move the comment up a line